packages/test/test-end-to-end-tests/src/test/dataMigration.spec.ts

```diff
@@ -209,13 +226,30 @@ describeNoCompat("HotSwap", (getTestObjectProvider) => {
 
 		await provider.ensureSynchronized();
 
+		// Get premigration sequence number
+		const preMigrationSeqNumber1 = container1.deltaManager.lastKnownSeqNumber;
+		const preMigrationSeqNumber2 = container2.deltaManager.lastKnownSeqNumber;
+
 		// Hot swap
 		shim1.submitMigrateOp();
 
 		// TODO: shim1.on("migrated", () => { ... });
 		// TODO: shim2.on("migrated", () => { ... });
 		await provider.ensureSynchronized();
 
+		// Verify that no ops were generated during migration.
```

Comment:
I don't think it makes sense for us to test that no ops were sent if we're not enforcing it.  With the current plan nothing stops the customer from continuing to edit data (either in the tree or some other DDS), there's just an expectation that further edits to this specific tree won't make it through the migration.

Reply:
Sounds like we should enforce it. If we have ops generated every time a container generates a migrate op, that will lead to some unknown/unpredictable behavior. I can't predict what will happen if you try to generate a move, where you let's say move some data from one place to another, and generate two ops for it. This invariant expects that no ops will be generated. Putting some asserts may be the right thing here to prevent customers from hurting themselves.

Reply:
[AB#5992](https://dev.azure.com/fluidframework/internal/_workitems/edit/5992)